src/pages/news.tsx

```diff
@@ -44,19 +40,19 @@ InternalNews.getLayout = (page: React.ReactNode) =>
   withPageLayout(
     <div>
       <h1>Internal News</h1>
-      <BreadcrumbBar>
-        <Breadcrumb>
-          <BreadcrumbLink<NavLinkProps> asCustom={NavLink} href="/">
-            Service portal home
-          </BreadcrumbLink>
-        </Breadcrumb>
-        <Breadcrumb>
-          <BreadcrumbLink<NavLinkProps> asCustom={NavLink} href="/about-us">
-            About us
-          </BreadcrumbLink>
-        </Breadcrumb>
-        <Breadcrumb current>Internal News</Breadcrumb>
-      </BreadcrumbBar>
+      <BreadcrumbNav
+        navItems={[
+          {
+            path: '/',
+            label: <Breadcrumb>Service portal home</Breadcrumb>,
+          },
+          {
+            path: '/news',
+            label: <Breadcrumb>Internal News</Breadcrumb>,
```

Comment:
Question: Looks like the original navigation in this file was for `About us` should this be `About us` too? Though after looking at the pages in the browser looks like internal news isn't linked yet in the menu so maybe this isn't something we need to worry about right now.

Reply:
Ah I missed that. I don't think it would hurt to add that in.